app/web/features/dashboard/Hero/HeroLinks.tsx

```diff
@@ -0,0 +1,97 @@
+import { Link as MuiLink, ThemeProvider, Typography } from "@material-ui/core";
+import classNames from "classnames";
+import StyledLink from "components/StyledLink";
+import { DASHBOARD } from "i18n/namespaces";
+import { useTranslation } from "next-i18next";
+import { useState } from "react";
+import { routeToProfile } from "routes";
+import makeStyles from "utils/makeStyles";
+
+import CommunitiesDialog from "../CommunitiesDialog";
+import useHeroBackgroundTheme from "./useHeroBackgroundTheme";
+
+const useStyles = makeStyles((theme) => ({
+  linksContainer: {
+    display: "flex",
+    flexWrap: "wrap",
+    rowGap: theme.spacing(2),
+    columnGap: theme.spacing(4),
+    justifyContent: "center",
+    margin: theme.spacing(4, 0),
+  },
+  tabAppearance: {
+    position: "relative",
+    paddingBottom: theme.spacing(1),
+    "&::after": {
+      content: '""',
+      position: "absolute",
+      bottom: 0,
+      left: "50%",
+      transform: "translateX(-50%)",
+      width: 40,
```

Comment:
Interesting! It can make sense yeah.

I think I prefer it to be the same width, so when hovering the other links, the bars that show are the same width. I personally prefer that, but I don't have strong feelings at all. Let me know if you'd prefer to change it to a percentage


https://user-images.githubusercontent.com/1557348/168542063-a35dc5f4-65af-4d34-a451-da1bf1d00b81.mov

Reply:
sounds good!